docs/search/content_type_search_reference/content_type_criteria.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Content Type Search Criteria help define and fine-tune search queries for content types.
+page_type: reference
+month_change: true
+---
+
+# Content Type Search Criteria reference
+
+Content Type Search Criteria are only supported by Content Type Search (`ContentTypeService::findContentTypes`).
+
+| Criterion | Description |
+|-------|-------------|
+| [ContainsFieldDefinitionId](api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-Values-ContentType-Query-Criterion-ContainsFieldDefinitionId.html) | Matches content types that contain a field definition with the specified ID. |
+| [ContentTypeGroupId](api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-Values-ContentType-Query-Criterion-ContentTypeGroupId.html) | Matches content types by their assigned group ID. |
```

Comment:
`ContentTypeGroupName` criterion is missing

Reply:
ContentTypeGroupName has just been merged today and will not be released until late November (?). I will add it as a follow up to this PR. We can update the code sample to include it, too.